packages/app/pages/Api/economicResource.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * This exports a React element which displays a list of all economic resources,
+ * and provides a section to select a single economic resource out of that list
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-02-08
+ */
+
+import * as React from "react";
+import "./api.css";
+import getAllEconomicResources from "../../../ui-bindings/EconomicResource/getAllEconomicResources";
+import { concatArray } from "./agent";
```

Comment:
I don't like this dependency on Agent. If the function is a utility, I would define it in some utility location. Having it linked from Agent leads to problems if the function is ever changed to better suit Agent because it is not clear that the function is used in other locations.

Reply:
resolved